src/cascadia/TerminalApp/MinMaxCloseControl.xaml

```diff
@@ -10,12 +10,283 @@
+                                                <ObjectAnimationUsingKeyFrames Storyboard.TargetName="ClosePath" Storyboard.TargetProperty="Stroke">
+                                                    <DiscreteObjectKeyFrame KeyTime="0" Value="{StaticResource CloseStrokeBrushPointerOver}" />
+                                                </ObjectAnimationUsingKeyFrames>
+                                            </Storyboard>
+                                        </VisualState>
+
+                                        <VisualState x:Name="Pressed">
+
+                                            <Storyboard>
+                                                <ObjectAnimationUsingKeyFrames Storyboard.TargetName="ButtonBaseElement" Storyboard.TargetProperty="Background">
+                                                    <DiscreteObjectKeyFrame KeyTime="0" Value="{StaticResource CloseBackgroundBrushPressed}" />
+                                                </ObjectAnimationUsingKeyFrames>
+                                                <ObjectAnimationUsingKeyFrames Storyboard.TargetName="ClosePath" Storyboard.TargetProperty="Stroke">
+                                                    <DiscreteObjectKeyFrame KeyTime="0" Value="{StaticResource CloseStrokeBrushPressed}" />
+                                                </ObjectAnimationUsingKeyFrames>
+                                            </Storyboard>
+                                        </VisualState>
+
+                                        <VisualState x:Name="Disabled" />
+
+                                    </VisualStateGroup>
+
+                                </VisualStateManager.VisualStateGroups>
+
+                                <Path
+                                    x:Name="ClosePath"
+                                    StrokeThickness="1"
+                                    Stroke="{ThemeResource CaptionButtonStrokeBrush}"
+                                    Data="M 0 0 L 10 10 M 10 0 L 0 10"
+                                    Stretch="Fill"
+                                    UseLayoutRounding="False"
+                                    Width="11"
+                                    Height="11"
+                                    StrokeEndLineCap="Square"
+                                    StrokeStartLineCap="Square" />
+                            </Border>
+
+                        </ControlTemplate>
+                    </Setter.Value>
+                </Setter>
+            </Style>
+        </ResourceDictionary>
+    </StackPanel.Resources>
+
+    <Grid x:Name="Content"></Grid>
+    <Border Height="36.0" MinWidth="160.0" x:Name="DragBar" Background="{ThemeResource ApplicationPageBackgroundThemeBrush}" DoubleTapped="DragBar_DoubleTapped"/>
+    <Button Height="36.0" Width="45.0" x:Name="Minimize" Style="{StaticResource CaptionButton}" Click="Minimize_Click">

```

Comment:
Accessibility: missing tooltip and accessible names

Reply:
The caption buttons don't seem to have tooltips, but I've added accessible names.